### DISCUSSION TITLE
付费购买专业版之后没有续期

### DISCUSSION THREAD

**Discussion by allenchenhz** [getlantern/lantern-forum-cn#608] [General]
@Mike777Khoo
邮箱541831873@qq.com
![12eab0b1b6fb2cc73937ed670a00fd3](https://github.com/getlantern/lantern-forum-cn/assets/173217561/2c771375-68c4-4e2c-b449-d9662f4f3e04)

**Comment by allenchenhz** [comment]
@Mike777Khoo

**Comment by allenchenhz** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
@allenchenhz 您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.
同时也请您重新登录. 之前的问题应该修好了.

**Reply by allenchenhz** [reply]
已发送但仍然没有解决。请问如何重新登录

**Comment by allenchenhz** [comment]
已发送至链接。但不知如何重新登录。目前的问题仍未解决

Mike K ***@***.***> 于2024年6月21日周五 02:26写道：

**Comment by allenchenhz** [comment]
还是没有任何消息或者解决呀
